packages/fxa-content-server/app/scripts/templates/sms_send.mustache

```diff
@@ -3,7 +3,15 @@
       <div class="error"></div>
     </header>
     <section class="send-sms">
-      <div class="graphic {{graphicId}}">{{#t}}Success{{/t}}</div>
+      <div class="graphic {{graphicId}}" role="img" aria-label="{{#t}}Connect another device{{/t}}"></div>
```

Comment:
I'm curious if this needs to be a `div` at all, and not just an `img`. If the goal is to make it not interactive we could apply `user-select: none;`. A little out of scope so feel free to disregard.

Reply:
Yeah you're right, this should just be an image. That reminds me of #3951 where we have some links as buttons. It'd require some refactoring since we grab the image class server-side and apply it here as a BG-image, I know some folks prefer BG images to `img` but if we do it that way, cases like this should have a `role="img"` and an `aria-label` for alt text and we're doing this in other places without the role/aria-label. Either way, I'll create a follow up issue with an `a11y` label for that.